docs/reference/filters.md

```diff
@@ -16,6 +16,41 @@ Example route with a match all, 2 filters and a backend:
 all: * -> filter1 -> filter2 -> "http://127.0.0.1:1234/";
 ```
 
+## backendIsProxy
+
+Notifies the proxy that the backend this request is going to be sent to is also
+proxy. This means that the request is going to be sent the usual proxy way:
```

Comment:
The route can be used as a proxy route using `CONNECT` as far as I understand from the code. 
Can you add in the documentation that it reflects this and maybe also which proxy protocols we support with this?
https://golang.org/pkg/net/http/#Transport. Do I understand correctly that this would also support socks5?

Reply:
I updated the documentation in the latest commit. Not sure if that's what you were looking for, if not can you please explain in more detail?

Reply:
Thanks docs look good to me!